Makefile

```
@@ -213,11 +213,11 @@ bundle: manifests operator-sdk kustomize ## Generate bundle manifests and metada
 	$(OPERATOR_SDK) generate kustomize manifests -q --package $(BUNDLE_PACKAGE)
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
 	$(KUSTOMIZE) build config/manifests | $(OPERATOR_SDK) generate bundle -q --overwrite --package $(BUNDLE_PACKAGE) --version $(VERSION) $(BUNDLE_METADATA_OPTS)
-	sed -i '/creationTimestamp/d' ./bundle/manifests/*.yaml
+	./hack/inplace_sed.sh '/creationTimestamp/d' ./bundle/manifests/*.yaml
```

The -i.bak seems working on both BSD and GNU systems:
```suggestion
	sed -i.bak '/creationTimestamp/d' ./bundle/manifests/*.yaml && rm ./bundle/manifests/*.bak
```